src/content/docs/features/single-instance.mdx

```diff
@@ -1,14 +1,80 @@
 ---
 title: Single Instance
-description: Ensure a single instance of your tauri app is running.
+description: Plugin to ensure that a single instance of your tauri app is running at a time.
 ---
 
-import Stub from '@components/Stub.astro';
 import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <PluginLinks plugin="single-instance" showJsLinks={false} />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/single-instance
-</Stub>
+Plugin to ensure that a single instance of your tauri app is running at a time.
+
+
+## Install
+
+_This plugin requires a Rust version of at least **1.75**_
+
+You can install the Core plugin by adding the following to your `Cargo.toml` file:
+
+<Tabs>
+  <TabItem label="crates.io">
+```toml title="src-tauri/Cargo.toml" ins={2}
+[dependencies]
+tauri-plugin-single-instance = "2.0.0-beta"
+```
+  </TabItem>
+  <TabItem label="Git">
+
+```toml title="src-tauri/Cargo.toml" ins={2}
+[dependencies]
+tauri-plugin-single-instance = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+```
+  </TabItem>
+</Tabs>
+
+## Usage
+
+To use this plugin, you need to register the core plugin with Tauri:
+
+```rust title="src-tauri/src/main.rs" ins={4}
+fn main() {
+    tauri::Builder::default()
+        .plugin(tauri_plugin_shell::init())
+        .plugin(tauri_plugin_single_instance::init(|_, _, _| {}))
+        .invoke_handler(tauri::generate_handler![greet])
+        .run(tauri::generate_context!())
+        .expect("error while running tauri application");
+}
+```
```

Comment:
```suggestion
```rust title="src-tauri/src/lib.rs" ins={4}
#[cfg_attr(mobile, tauri::mobile_entry_point)]
pub fn run() {
    tauri::Builder::default()
        // Initialize the plugin
        .plugin(tauri_plugin_single_instance::init(|app, args, cwd| {}))
        .run(tauri::generate_context!())
        .expect("error while running tauri application");
}
```
```

Reply:
I guess `#[cfg_attr(mobile, tauri::mobile_entry_point)]` is not necessary for this example. Adapted other suggestions.

Thanks a lot for suggesting :blush:

Reply:
it's a different file too